index.html

```diff
@@ -57,6 +72,9 @@
         const app = document.getElementById("app");
         app.classList.add("container");
         document.body.appendChild(app);
+        const cards = document.createElement("div");
+        cards.classList.add("cards");
+        app.appendChild(cards);
```

Comment:
Next, create a new div with the class cards and update the existing JavaScript code to use it for the card elements. Add the following code after the app.classList.add("container"); line in the <script> section: